static/app/components/avatar/avatarList.tsx

```diff
@@ -1,14 +1,16 @@
 import {css} from '@emotion/react';
 import styled from '@emotion/styled';
 
+import TeamAvatar from 'sentry/components/avatar/teamAvatar';
 import UserAvatar from 'sentry/components/avatar/userAvatar';
 import {Tooltip} from 'sentry/components/tooltip';
 import {AvatarUser} from 'sentry/types';
+import {TeamParticipant} from 'sentry/types/group';
```

Comment:
I don't love using `TeamParticipant` here because it is specific to issues, whereas this is a common component used in many parts of the app